docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -1,39 +1,180 @@
+
+### Attribute: `instanceFilter`
+
+Specifies an [ECExpression](./ECExpressions.md#instance-filter) for filtering instances of ECClasses specified through the [`classes` attribute](#attribute-classes).
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.InstanceFilter.Ruleset]]
+```
+
+  |                | Result                                                                                                       |
+  | -------------- | ------------------------------------------------------------------------------------------------------------ |
+  | without filter | ![Example when selecting all instances](./media/contentinstancesofspecificclasses-with-instancefilter-1.png) |
+  | with filter    | ![Example when filtering instances](./media/contentinstancesofspecificclasses-with-instancefilter-2.png)     |
+
+### Attribute: `onlyIfNotHandled`
+
+> **Default value:** `false`
+
+Identifies whether we should ignore this specification if another specification was already handled (based on rule priorities and definition order). Should be used when defining a fallback specification.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.OnlyIfNotHandled.Ruleset]]
+```
+
+  | onlyIfNotHandled | Result                                                                                                         |
+  | ---------------- | -------------------------------------------------------------------------------------------------------------- |
+  | `true`           | ![Example when selecting all instances](./media/contentinstancesofspecificclasses-with-onlyifnothandled-1.png) |
+  | `false`          | ![Example when filtering instances](./media/contentinstancesofspecificclasses-with-onlyifnothandled-2.png)     |
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Defines the order in which specifications are handled - higher priority means the specifications is handled first. If priorities are equal, the specifications are handled in the order they're defined.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.Priority.Ruleset]]
+```
+
+![Example of using priority attribute](./media/contentinstancesofspecificclasses-with-priority.png)
+
+### Attribute: `relatedProperties`
+
+Specifications of [related properties](./RelatedPropertiesSpecification.md) which are included in the generated content.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.RelatedProperties.Ruleset]]
+```
+
+  |         | Result                                                                                                                    |
+  | ------- | ------------------------------------------------------------------------------------------------------------------------- |
+  | without | ![Example when doing normal property select](./media/contentinstancesofspecificclasses-with-relatedproperties-1.png)      |
```

Comment:
I think the example would look much better in property grid - it would show the actual related properties rather than a clickable link that we see in tables.